content/collections/api/en/taxonomy.md

```diff
@@ -1617,7 +1632,7 @@ new_user_property_value=VALUE&description=DESCRIPTION
 {{/partial:tabs}}
 
 {{partial:collapse name="Example: Update a user property"}}
-This example updates a user property called "user_type" to be named "subscription_type", adds a description of "The user's subscription type", and changes the property's data type to `string`. The user property is prefixed with `gp:` in the path because it's a custom user property. 
+This example updates a user property called "user_type" to be named "subscription_type", adds a description of "The user's subscription type", and changes the property's data type to `string`. The user property is prefixed with `gp:` in the path because it's a custom user property.
```

Comment:
```suggestion
This example updates the `user_type` user property to be named `subscription_type`, adds a description of "The user's subscription type", and changes the property's data type to `string`. The user property is prefixed with `gp:` in the path because it's a custom user property.
```